### PR TITLE
Fetching and listing iteration (fixes #708)

### LINKED ISSUE #708: Display list of iterations
This task is part of #647

### PR DESCRIPTION
**Pull request by sanbornsen**
For now it fetches the list of spaces and work with the first space in the list. Once we implement the shared service or anything similar to fetch the default space this temporary code for fetching spaces will be removed.

Space and iteration need to be created from the backend or using the API to test this feature.

### CODE REVIEW

File: src/app/iteration/iteration.component.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
                tooltip="{{(100 - barchatValue) + '%'}}">
           </div>
         </div>
-        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+        <!-- Lorem Ipsum is simply dummy text of the printing and typesetting industry. -->
```
Comment (vikram-raj) [inline src/app/iteration/iteration.component.html]: Don't we have description text now?

File: src/app/iteration/iteration.module.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { IterationService } from './iteration.service';
```
Comment (vikram-raj) [inline src/app/iteration/iteration.module.ts]: This import should be after angular imports.

File: src/app/iteration/iteration.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/iteration/iteration.component.html]: Not right now

File: src/app/iteration/iteration.module.ts (hunk 2 above)
Comment (sanbornsen) [inline src/app/iteration/iteration.module.ts]: Done

File: src/app/iteration/iteration.service.ts [hunk 3]
```diff
@@ -130,4 +130,21 @@ export class IterationService {
...
+
+  // Temporary space fetch
+  getSpaces() {
+    return this.http
+    .get(process.env.API_URL + 'spaces', { headers: this.headers })
+    .toPromise()
+    .then((response) => {
+      let spaces = response.json().data;
+      if (spaces.length) {
+        return spaces[0];
```
Comment (debloper) [inline src/app/iteration/iteration.service.ts]: I understand this is temporary fetch, but why are we only extracting/using the 1st space only?

**Pull request merged**